tests/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openeo/processes-validator",
```

Comment:
Is this package on npm? Can't find it: https://www.npmjs.com/search?q=%40openeo%2F